workflows.tf

```diff
@@ -62,7 +62,7 @@ resource "google_workflows_workflow" "project_setup" {
   region          = var.region
   description     = "Copies data and performs project setup"
   service_account = google_service_account.workflows_sa.email
-  source_contents = templatefile("${path.module}/assets/yaml/initial-workflow-project-setup.yaml", {
+  source_contents = templatefile("${path.module}/src/yaml/initial-workflow-project-setup.yaml", {
```

Comment:
Note: we'll want to open a bug to track moving these to template file location but we can merge in for now.